### ISSUE TITLE
Clarify the Goal of Koan

### ISSUE THREAD

**Issue by gregberns** [crazymykl/rust-koans#40]
Maybe its just me, but its not clear what this koan is supposed to accomplish? There are no hints as to what I need to change.

```
// Arrays can also be iterated through using a for loop
#[test]
fn for_loops() {
    let arr: [u64; 3] = [1, 2, 3];
    let mut y: u64 = 1;
    for x in &arr {
        assert!(*x == y);
    }
}
```
https://github.com/crazymykl/rust-koans/blob/266553469b0103e979480f1a6318b25e8750f728/src/koans/array.rs#L95-101

Thx for the project!

**Comment by gideonb234** [comment]
+1 I'm not sure what I'm meant to accomplish here either

I think make the values all match 1 but I'm not sure?

**Comment by AZanellato** [comment]
Really late, but just want to clarify that `y` is mut because you need to add 1 to it every loop.

@crazymykl Can clarify, but this is what I think.

**Comment by crazymykl** [comment]
Closed by #52.
